metadata-ingestion-modules/airflow-plugin/src/datahub_airflow_plugin/_config.py

```diff
@@ -50,6 +50,8 @@ class DatahubLineageConfig(ConfigModel):
 
     disable_openlineage_plugin: bool = True
 
+    render_templates: bool = True
```

Comment:
It is fine as it is but I think it would make sense to add descriptions here as well which makes it easier to read what these config properties do